setup.py

```diff
@@ -1,20 +1,22 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
-from telomerecat.version import __version__
 
 setup(
   name="telomerecat",
+  version='3.4.1',
   description="Telomere Computational Analysis Tool",
-  version=__version__,
+  url='https://github.com/cancerit/telomerecat',
   author="JHR Farmery",
   license="GPL",
   python_requires='>= 3.6',
   author_email="cgphelp@sanger.ac.uk",
   packages=["telomerecat"],
   package_dir={"telomerecat": "telomerecat"},
-  install_requires=["parabam>=2.3.0", "numpy", "pysam", "pandas"],
+  install_requires=["parabam>=2.3.2", "numpy", "pysam", "pandas", "click"],
```

Comment:
Thanks for sorting the release file.  The install requires technically will allow 2.3.0, but it should be updated to 3.0.0 really.

How virtualenv is installed has changed.  We shouldn't include that in the user docs, other than to say they need it... it may not be user installable for some (due to dependencies).

Reply:
docs also updated